dataflow/src/main/java/org/checkerframework/dataflow/cfg/playground/BusyExpressionPlayground.java

```diff
@@ -6,29 +6,32 @@
 import org.checkerframework.dataflow.busyexpr.BusyExprStore;
 import org.checkerframework.dataflow.busyexpr.BusyExprTransfer;
 import org.checkerframework.dataflow.cfg.visualize.CFGVisualizeLauncher;
+import org.checkerframework.dataflow.cfg.visualize.CFGVisualizeOptions;
 
 /** The playground for busy expression analysis */
 public class BusyExpressionPlayground {
 
     /**
      * Run busy expression analysis playground on a test file and print the CFG graph
      *
-     * @param args input arguments
+     * @param args command-line arguments
      */
     public static void main(String[] args) {
 
-        /* Configuration: change as appropriate */
-        String inputFile =
-                "dataflow/manual/examples/BusyExprSimple.java"; // input file name and path
-        String outputDir = "."; // output directory
-        String method = "test"; // name of the method to analyze
-        String clazz = "Test"; // name of the class to consider
+        /** Parse the arguments. */
+        CFGVisualizeOptions config = CFGVisualizeOptions.parseArgs(args);
 
         // Run the analysis and create a PDF file
         BusyExprTransfer transfer = new BusyExprTransfer();
         BackwardAnalysis<UnusedAbstractValue, BusyExprStore, BusyExprTransfer> backwardAnalysis =
                 new BackwardAnalysisImpl<>(transfer);
         CFGVisualizeLauncher.generateDOTofCFG(
-                inputFile, outputDir, method, clazz, true, true, backwardAnalysis);
+                config.getInput(),
+                config.getOutput(),
+                config.getMethod(),
+                config.getClas(),
+                true,
```

Comment:
Throughout, shouldn't these `true, true` be replaced with `config.isPDF(), config.isVerbose()`? Check the order of the two booleans.

Reply:
Also, `config.isString()` isn't used here. How about we add a new method to `CFGVisualizeLauncher` that takes `transfer`, `analysis`, and `config` and then does the right thing?

Reply:
Understood, I will make the changes! My mistake :)

Reply:
You can simply commit the suggested edits directly on GitHub - add them to a batch and then commit them. When you edit more locally, remember to pull in from remote first.

Reply:
Is `trasnfer` a parameter relevant to the methods inside `CFGVisualizeLauncher`? I was thinking of having a method called `performAnalysis` that takes in a `CFGVisualizeOptions` config object as well as an `analysis` object. In the playgrounds, the `transfer` object is used to generate the `analysis`. My current declaration for the function is:
```
public static void performAnalysis(
	    CFGVisualizeOptions config,
	    @Nullable Analysis<V, S, T> analysis
)
```

Reply:
Yes, config and analysis should be enough.